src/pages/dongmin/Main/MainRight/MainRight.scss

```
@@ -0,0 +1,168 @@
+
+  .profileCnt {
+    display: flex;
+    align-items: center;
+    height: 28px;
+    overflow: hidden;
+    .profileImg {
+      width: 25px;
+      height: 25px;
+      border-radius: 50%;
+      margin-right: 5px;
+    }
+    .profileText li {
+      &:nth-child(1) {
+        font-size: 12px;
+      }
+      &:nth-child(2) {
+        font-size: 6px;
+      }
+    }
+  }
+
+  .storyWrap {
+    position: relative;
+    border: 2px solid rgb(245, 245, 245);
+    height: 200px;
+    padding: 8px;
+    background-color: white;
+    font-size: 10px;
+    overflow: scroll;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+
+    .showAll {
+      top: 5px;
+      right: 5px;
+      position: absolute;
+      font-size: 10px;
+      font-weight: 700;
+    }
+
+    .storyLi {
+      .sideProfileCnt {
+        display: flex;
+        align-items: center;
+        height: 35px;
+        overflow: hidden;
+
+        .sideProfileImg {
+          width: 30px;
+          height: 30px;
+          margin-right: 7px;
+          border-radius: 50%;
+        }
+
+        .sideProfileText {
+          display: block;
+          overflow: hidden;
+
+          li {
+            &:nth-child(1) {
+              font-size: 13px;
+              margin: 0;
+            }
+
+            &:nth-child(2) {
+              font-size: 10px;
+              margin: 0;
+              overflow: hidden;
+              white-space: nowrap;
+              text-overflow: ellipsis;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  .recommendWrap {
+    position: relative;
+    border: 2px solid rgb(245, 245, 245);
+    height: 200px;
+    padding: 8px;
+    background-color: white;
+    font-size: 10px;
+    overflow: scroll;
+
+    &::-webkit-scrollbar {
+      display: none;
+    }
+
+    .showAll {
+      top: 5px;
+      right: 5px;
+      position: absolute;
+      font-size: 10px;
+      font-weight: 700;
+    }
+
+    .storyLi {
+      .sideProfileCnt {
+        display: flex;
+        align-items: center;
+        height: 35px;
+        overflow: hidden;
+
+        .sideProfileImg {
+          width: 30px;
+          height: 30px;
+          margin-right: 7px;
+          border-radius: 50%;
+        }
+
+        .sideProfileText {
+          display: block;
+          overflow: hidden;
+
+          li {
+            &:nth-child(1) {
+              font-size: 13px;
+              margin: 0;
+            }
+
+            &:nth-child(2) {
+              font-size: 10px;
+              margin: 0;
+              overflow: hidden;
+              white-space: nowrap;
+              text-overflow: ellipsis;
+            }
+          }
+        }
+      }
+    }
+  }
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 30px;
+    opacity: 0.5;
+    li {
+      float: left;
+      margin: 0;
+      font-size: 11px;
+
+      &::after {
+        content: '•';
+      }
+
+      &:last-child::after {
+        content: '';
+      }
```

how about
```suggestion
      &:not(:last-child)::after {
        content: '•';
      }
```

css [not 연산자](https://developer.mozilla.org/ko/docs/Web/CSS/:not)

---

나름 고심 끝에 구현하고 뭔가 깔끔하지 않아서 찝찝했는데 not 연산자라는 게 있었군요 ㅎ.ㅎ; 수정했습니다!